qa/L0_sequence_batcher/test.sh

```diff
@@ -183,6 +183,17 @@ export USE_SINGLE_BUFFER
 #   models4 - four instances with batch-size 1
 rm -fr *.log  models{0,1,2,4} queue_delay_models && mkdir models{0,1,2,4} queue_delay_models
 
+# Search BACKENDS to determine if a backend should be tested
+function should_test_backend() {
```

Comment:
This might make sore sense to be put in the `utils.sh` file for other tests

Reply:
I was thinking the same thing. Of course, this function will be useless if developers don't know about it and use it properly. If you think it makes sense to add a function like this, we can merge it and I can make an announcement about it.

Reply:
I'm not a big fan of moving something for functional programming aside, and particularly in our case where we test cross OS and cross-platform. 
I have no idea about this function, but not everything should come out of `utilis.sh`, and for reproducing/testing purpose suppose the more simple and straight forward solution the better.
Note that any problem with those functions from `utils.sh` will impact all others tests that are uses it.

Reply:
In a sense @mc-nv I believe you're right. If people don't know about this function, we'll start observing mixed use-cases which is a mess (think about kill/wait vs. calling kill_server). I'll keep the function here if we're all in agreement.

Reply:
Fine by me